### DISCUSSION TITLE
Proposal: "Beef Up" docs, issue templates, etc.

### DISCUSSION THREAD

**Discussion by cjpillsbury** [muxinc/media-chrome#72] [Proposals]
As media-chrome starts moving to a more "official" project, we should probably start making the non-code aspects of the repository a bit more "polished" as well. By way of example, the [mux-elements](https://github.com/muxinc/elements) has added things like:
* [Contributing Docs and Badges](https://github.com/muxinc/elements#contribution)
    * Including a [Code of Conduct](https://github.com/muxinc/elements/blob/main/CODE_OF_CONDUCT.md)
* [npm package badges](https://github.com/cjpillsbury/elements/tree/docs/badges/packages/mux-video#readme)
* [Issue Templates](https://github.com/muxinc/elements/tree/main/.github/ISSUE_TEMPLATE)
* [Additional Labels](https://github.com/muxinc/elements/labels)
* [husky pre-commit hooks](https://github.com/muxinc/elements/blob/main/.husky/pre-commit)
* [standard .gitignore](https://github.com/muxinc/elements/blob/main/.gitignore)

**Comment by cjpillsbury** [comment]
fyi: for now, npm badges are in pending PR here: https://github.com/muxinc/elements/pull/35/commits/793d48919f184854c596564b448b7b7ed142193f

**Comment by dylanjha** [comment]
💯  I'm in favor of this, the setup on elements looks really good IMO

**Comment by heff** [comment]
Sounds good to me. I'm not familiar with husky pre-commit hooks.

**Comment by dylanjha** [comment]
Husky pre-commit hooks are most often used to run `yarn lint` anytime you do `git commit` -- it prevents developers from pushing up commits that will fail at the linting step
